sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -470,6 +472,13 @@ public static partial class MultipartResponse
         public static Azure.Response[] Parse(Azure.Response response, bool expectCrLf, System.Threading.CancellationToken cancellationToken) { throw null; }
         public static System.Threading.Tasks.Task<Azure.Response[]> ParseAsync(Azure.Response response, bool expectCrLf, System.Threading.CancellationToken cancellationToken) { throw null; }
     }
+    public partial class ProcessingContext
+    {
+        internal ProcessingContext() { }
+        public int AttemptNumber { get { throw null; } }
```

Comment:
Might make sense to call this "RetryAttempt" so it's clear it's related to retries, with the type being renamed s.t. it doesn't name retries specifically.

Reply:
Changed to RetryNumber.